### PR TITLE
Add support for an ip set rule

### PR DESCRIPTION
**Pull request by mark-painter**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -34,6 +36,14 @@ output "web_acl_id" {
 
 locals {
     name_underbar = replace(var.name, "-", "_")
+
+    ip_sets_rule = var.ip_set != null ? [ {
+      name       = "${var.ip_set_action}_ips"
```
Comment (luckyvalentine) [inline main.tf]: I can't be certain, but I feel that having a "IP set rule" with the name of "block_ips" or "allow_ips" might collide when the same rules get used for dev, qa, staging and prod.

If they're tied to a particular WAF "namespace" then it should be fine as the WAF has the stage in it.  Thoughts?

Comment (luckyvalentine) [reply main.tf]: test

Comment (mark-painter) [reply main.tf]: The rules do need to be unique per waf, but I don't believe they need to be among all wafs

Comment (mark-painter) [reply main.tf]: I've updated to have a more specific name

**Review by luckyvalentine — APPROVED** [review]
nice.  LGTM.

**Pull request merged**